openapi/endpoints/accounts/models/create-cardholder-account-request.yaml

```diff
@@ -0,0 +1,22 @@
+type: object
+required:
+  - type
+  - partnerAccount
```

Comment:
⚠️ Important: Should this be `partnerAccountId`?

Reply:
absolutely, good catch